src/app/filters/filter-field.ts

```diff
@@ -12,6 +12,7 @@ import { FilterQuery } from './filter-query';
 export class FilterField {
   id?: string;
   placeholder?: string;
+  primaryQueries?: FilterQuery[];
```

Comment:
primaryQueries is the first section of the list which is not filterable. Currently this is only added for typeahead section.
<img width="442" alt="screen shot 2017-04-19 at 12 20 36 am" src="https://cloud.githubusercontent.com/assets/2410474/25147711/13d09286-2496-11e7-9a98-034d3e2fa666.png">